Daniel-Bennett777/Buildo-API
# fixtures and models done

Added in the Erd references and the actual models which are apparent in our erd schema [https://dbdiagram.io/d/BuilDO-65666c723be1495787ec98d1](url) 
**To Test**
- git checkout main
- git fetch
- git checkout DL

check to see that there are fixtures and models that coordinate with the erd. check models for the models, fixtures for the fixtures.
